docs/getting-started/concepts.md

```diff
@@ -49,3 +49,46 @@ Iter8 is written in `go` and builds on a few awesome open source projects includ
 - [Fortio](https://github.com/fortio/fortio)
 - [ghz](https://ghz.sh)
 - [plotly.js](https://github.com/plotly/plotly.js)
+
+## AutoX
+
+AutoX, short for "auto experimentation", allows Iter8 to detect changes to your Kubernetes resources objects and automatically start new experiments, allowing you to test your applications as soon as you push out a new version.
+
+To configure AutoX, you will need to specify a set of experiment groups and, for each group, the Kubernetes resource object (a.k.a. trigger object) that you expect AutoX to watch and one or more experiments to be performed in response to new versions of this object.
+
+The trigger object is specified by providing the name, namespace, and the group-version-resource (GVR) metadata of the trigger object.
+
+See the following example:
```

Comment:
Does it make sense too provide a second example, with two groups (e.g., a grpc app also getting tested, with its own `group`), each with its own experiment?